src/endpoints/user.rs

```diff
@@ -7,20 +7,31 @@ use chrono::{DateTime, Utc};
 /// Gets information about a user
 /// https://api.cloudflare.com/#user-user-details
 
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Eq)]
+pub struct Organization {
```

Comment:
It could be useful to add a description of these fields - you can just copy and paste the descriptions from https://api.cloudflare.com/#user-properties (click organizations > show definition)